### ISSUE TITLE
Issue with onComplete callback in CameraFlyTo

### ISSUE THREAD

**Issue by yfilatov** [darwin-education/resium#420]
Hi, 

The onComplete callback doesn't executes after camera movement ends.

Usage:
```
<CameraFlyTo
            orientation={orientationCartesian}
            destination={destinationCartesian}
            duration={1}
            onComplete={()=>{console.log("Fly ends")}}
        />
```

Movement ends successfully but no callback.

Resium:  "version": "1.11.0",
Cesium:  "version": "1.70.1",

**Comment by rot1024** [comment]
Thank you for reporting. It will be fixed in new version 1.12.

**Comment by rot1024** [comment]
v1.12 has been released. Please try it and reopen this issue if the bug persists.
